src/Http/AbstractHttpMessage.php

```diff
@@ -51,11 +53,12 @@ public function addHeader(string $name, string $value)
 
     /**
      * @param string $header
+     * @param mixed  $default
      * @return string
      */
-    public function getHeader($header)
+    public function getHeader(string $header, $default = null)
```

Comment:
@valanz you can put whatever you want (otherwise it would be `string` type hinted). It's a bit tricky but you could imagine an object implementing `__toString` method... ? Nevermind it's ok like this.